src/Datatypes/datatypes.ts

```diff
@@ -2,7 +2,7 @@ export type Exercise = {
   id: string;
   title: string;
   description: string;
-  queries: Query[];
+  queries?: Query[];
```

Comment:
Why can queries by undefined? Instead of undefined an empty array of queries should be used.